examples/onprem_demo/README.md

```diff
@@ -0,0 +1,70 @@
+
+1. Create the Terraform variables file.
+    - In your favorite text editor, edit and save the following variables in `examples/onprem_demo/terraform.tfvars`:
+        - `vpc_sc_project_id`: the unique ID for the VPC Service Control project that is going to be created.  Use lowercase letters and a 6 digit random number only (no spaces).
+        - `onprem_project_id`: the unique ID for the on-prem project that is going to be created.  Use lowercase letters and a 6 digit random number only (no spaces).
+        - `credentials_path`: the name of the service account secret key file you downloaded.  Relative path is fine if you saved it in the `onprem_project` directory.
+        - `organization_id`: run `gcloud organizations list` to get this **number**.
+        - `billing_account_id`: the billing account that the project will charge to.
+        - `region`: region where the GCP resources will be created. 
+        - `vpn_shared_secret`: any random text string can go here.
+        - `access_policy_name`: Get this value by running the following two commands:
+            - `gcloud organizations list` to get the Organization ID
+            - `gcloud access-context-manager policies list --organization <Organization ID>` 
+2. From a command line terminal:
+    - `cd examples/onprem_demo`
+    - `terraform plan`
+    - `terraform apply` and type "yes" when it prompts you to create the resources.  Might take a few minutes to create the projects (should be less than 5 minutes).
+    - If you get error messages about the Compute Engine API or Cloud DNS API needing to be enabled, wait a minute and try running `terraform apply` again.  If you get error messages about forwarding rules and VPN tunnel creation, wait a minute and try again.
+
+
+To gain access to a VM in the VPC Service Control protected environment:
+
+- Set the Windows password for the jumphost in the on-prem project, and download the .rdp file
+- Set the Windows password for the VPC SC Windows VM
+- RDP into the jumphost using its public IP
+- Once you're in the jumphost, click Start -> Remote Desktop, and then RDP into `10.7.0.2` (the internal IP of the VM in the VPC Service Controls perimeter).  So you RDP into the jumphost, and then you RDP through the VPN tunnel into your cloud VM.  
+- Play around with VPC Service Controls.
+- Remember to delete all resources when done.  Easiest way to do this is to run `terraform destroy`.
+
+When you're done with this demo, delete the service account keys and delete the service account altogether from the Cloud Console.
```

Comment:
Adding these placeholders are necessary to create documentation for your example.
Once you've added them, install terraform-docs and run `make generate_docs` command from the root folder as described [here](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls#autogeneration-of-documentation-from-tf-files)


```suggestion
When you're done with this demo, delete the service account keys and delete the service account altogether from the Cloud Console.

[^]: (autogen_docs_start)
[^]: (autogen_docs_end)
```

Reply:
Is there any way we can avoid this?  This seems like needless complexity.  Much easier for me to simply type some text in a README than to figure out what's going on in that autogen link.

Reply:
No unfortunately this is using terraform-docs is our standard way of documenting all modules and examples in modules.
If you're having trouble making it work let me know. It shouldn't be harder than downloading terraform-docs, adding it to your path and running `make generate_docs`

Reply:
Done.